<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="loadImageBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
background-color: #d35400;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;</string>
    </property>
    <property name="text">
     <string>Load image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImageBtn">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
color: #fff;
background-color: #2ecc71;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}
:disabled {
color: #fff;
background-color: #dddddd;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}</string>
    </property>
    <property name="text">
     <string>Save image</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>411</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>401</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="contururiCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Contours</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="poligoaneCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Polygons</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cercuriCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Circles</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultImg">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>401</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImageBtn_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">:enabled {
color: #fff;
background-color: #5a28ef;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}
:disabled {
color: #fff;
background-color: #dddddd;
border-color: #357ebd;
font-size: 14px;
font-weight: normal;
border-radius: 4px;
text-align: center;
vertical-align: middle;
border: 1px solid transparent;
padding: 10px 16px;
}</string>
    </property>
    <property name="text">
     <string>Process image</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
